### ISSUE TITLE
Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/public-campaign-site#7]
Copy of [midd-frontend issue #712](https://github.com/middlebury/midd-frontend/issues/712).

**Comment by poojagunturu96** [comment]
Pages this change affects:

1. Background color in menu bar and campaign stats at the bottom of the page
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-unrestricted-giving.html
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-experience.html
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-access.html

2. Background color in left hand menu
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-basic-access.html
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-basic-experience.html
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-basic-unrestricted-giving.html

3. Pull quote component:
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-basic-content.html

4. 'Access' and 'Experience' cards on homepage
    - https://public-campaign-site-rewjw5ic5-middlebury.vercel.app/campaign-home.html
